x/feemarket/spec/01_concepts.md

```diff
@@ -0,0 +1,121 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## EIP-1559: Fee Market
+
+[EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md) describes a pricing mechanism
+that was proposed on Ethereum to improve to calculation of transaction fees.
+It includes a fixed-per-block network fee that is burned
+and dynamically expands/contracts block sizes to deal with peaks of network congestion.
+
+Before EIP-1559 the transaction fee is calculated with
+
+```
+fee = gasPrice * gasLimit
+```
+
+where `gasPrice` is the price per gas and `gasLimit` describes the amount of gas required to perform the transaction.
+The more complex operations a transaction requires, the higher the gas limit
+(see [Executing EVM bytecode](https://docs.evmos.org/modules/evm/spec/01_concepts.html#executing-evm-bytecode)).
+To submit a transaction, the signer needs to specify the `gasPrice`.
+
+With EIP-1559 enabled, the transaction fee is calculated with
+
+```
+fee = (baseFee + priorityTip) * gasLimit
+```
+
+where `baseFee` is the fixed-per-block network fee per gas
+and `priorityTip` is an additional fee per gas that can be set optionally.
+Note, that both the base fee and the priority tip are gas prices.
+To submit a transaction with EIP-1559, the signer needs to specify the `gasFeeCap`,
+which is the maximum fee per gas they are willing to pay in total.
+Optionally, the `priorityTip` can be specified,
+which covers both the priority fee and the block's network fee per gas (aka: base fee).
+
+::: tip
+The Cosmos SDK uses a different terminology for `gas` than Ethereum.
+What is called `gasLimit` on Ethereum is called `gasWanted` on Cosmos.
+You might encounter both terminologies on Evmos since it builds Ethereum on top of the SDK,
+e.g. when using different wallets like Keplr for Cosmos and Metamask for Ethereum.
+:::
+
+## Base Fee
+
+The base fee per gas (aka base fee) is a global gas price defined at the consensus level.
+It is stored as a module parameter
+and is adjusted at the end of each block
+based on the total gas used in the previous block
+and gas target (`block gas limit / elasticity multiplier`):
+
+- it increases when blocks are above the gas target,
+- it decreases when blocks are below the gas target.
+
+Instead of burning the base fee (as implemented on Ethereum),
+the `feemarket` module allocates the base fee
+for regular [Cosmos SDK fee distribution](https://docs.evmos.org/modules/distribution/).
+
+## Priority Tip
+
+In EIP-1559, the `max_priority_fee_per_gas`, often referred to as `tip`,
+is an additional gas price that can be added to the `baseFee` in order to incentivize transaction prioritization.
+The higher the tip, the more likely the transaction is included in the block.
+
+Until the Cosmos SDK version v0.46, however, there is no notion of transaction prioritization.
+Thus, the tip for an EIP-1559 transaction on Ethermint should be zero
```

Comment:
Should this be changed to Evmos as well? It will only be visible to users on the Evmos docs
```suggestion
Thus, the tip for an EIP-1559 transaction on Evmos should be zero
```

Reply:
yes